code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +92,50 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	var/ref_client = "\ref[src]"
+	for(var/datum/adminticket/T in admintickets)

```

Comment:
I'll be honest, I've been trying to figure out some way to make them into a istypeless loop and I just keep blanking.